### ISSUE TITLE
ICT.open: Send logo for stand

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna#248]
"We also offer you the possibility to print your logo on your demo table. The space on the table where the logo will be displayed, is 100x60cm (hxw). Please send your logo by Wednesday 9 March to deborah@spitz.nu."
What logo shall we send? DIANNA or eScience Center?

**Comment by elboyran** [comment]
I am not sure if the merging of the 2 logos looks good? Shall we submit the DIANNA logo only?
![netherlands-escience-center-DIANNA-logo](https://user-images.githubusercontent.com/3244249/156650671-2935c64f-69d6-4313-9a23-8a10fad09417.png)

**Comment by gcroci2** [comment]
I also tried merging them, taking inspiration from SURF logo:

![dianna_esc_logo](https://user-images.githubusercontent.com/55382553/156830470-9ed8d5e8-3704-445a-8236-330d83990cb1.png)

maybe it looks better with a smaller eScience logo?

**Comment by elboyran** [comment]
@gcroci2 that looks pretty nice! Any reason the BG is not pure white?

**Comment by gcroci2** [comment]
> 

Yes, I like it!! What do you mean by transparency? If you prefer a transparent background, I can easily do it (we won't have the fancy SURF-like shape but of course, it's okay as well). So something like this: 

![transp_logo](https://user-images.githubusercontent.com/55382553/157074404-802b53e2-8841-482d-b8cc-b7b7ecd5b5a7.png)

**Comment by elboyran** [comment]
I like the most the one with the SURF shape and white BG.

**Comment by elboyran** [comment]
Sent.

**Comment by cwmeijer** [comment]
![logowithnlesc_t_s](https://user-images.githubusercontent.com/6087314/157443983-9a912faf-e0db-48c2-85b4-932a15798c27.png)
For completeness, this is the logo I used for the video.
